src/services/ExperienceService.ts

```
@@ -0,0 +1,83 @@
+      }
+      
+      return res.json();
+    } catch (error) {
+      console.error("Error adding experience:", error);
+      throw error;
+    }
+  }
+
+  static async updateExperience(token: string, experienceId: number, experienceData: ExperienceRequestDTO) {
+    try {
+      const res = await fetch(`${this.BASE_URL}/experiences/${experienceId}`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)